### PR TITLE
Dependencies updates

### PR DESCRIPTION
**Pull request by jvega190**
- Update rxjs from 7.0.0 to 7.8.1
- Update redux-observable from 2.0.0 to 3.0.0 -rc.2
- Update @mui/lab from 5.0.0-alpha.155 to 5.0.0-alpha.156 https://github.com/craftercms/studio-ui/pull/3538
- Update @mui/material from 5.14.20 to 5.15.0 - https://github.com/craftercms/studio-ui/pull/3537

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,8 @@
     "nth-check": "^2.0.1",
     "@typescript-eslint/eslint-plugin": "^6.14.0",
     "@typescript-eslint/parser": "^6.14.0",
-    "fork-ts-checker-webpack-plugin": "^6.5.3"
+    "fork-ts-checker-webpack-plugin": "^6.5.3",
+    "rxjs": "^7.8.1"
```
Comment (rart) [inline package.json:69]: Is redux observable pulling another lower version? Why is this required?

**Review by rart — CHANGES_REQUESTED** [review]
I saw multiple resolutions of @mui/base and @mui/utils and stopped there — there might be others. As I mentioned, we need a single resolution for these libraries. Check and resolve the yarn.lock diffs.

**Comment by jvega190** [comment]
> I saw multiple resolutions of @mui/base and @mui/utils and stopped there — there might be others. As I mentioned, we need a single resolution for these libraries. Check and resolve the yarn.lock diffs.

Yeah I did fix it but the multiple resolutions 'reappeared' after the second version update in this PR. By running `yarn dedupe` again the resolutions got fixed

File: package.json (hunk 1 above)
Comment (jvega190) [reply package.json:69]: I was getting some errors due to '@craftercms/content' having rxjs v7.0.0. Should we update the sdk? and or keep this resolution?

Comment (rart) [reply package.json:69]: Update sdk

**Review by rart — APPROVED** [review]

**Comment by sumerjabri** [comment]
Will not merge without a ticket.

**Comment by rart** [comment]
> Will not merge without a ticket.

@sumerjabri the _tickets_ are the Dependabot PRs that have been assigned (like we've been doing lately with the stream of Dependabot PRs that need validation, consolidation or any kind of special handling)

**Comment by rart** [comment]
@jvega190 mui needs a bump already (see https://github.com/craftercms/studio-ui/pull/3555)

**Comment by sumerjabri** [comment]
https://github.com/craftersoftware/craftercms/issues/686
